### DISCUSSION TITLE
Collaborative notifications

### DISCUSSION THREAD

**Discussion by almereyda** [outline/outline#6489] [Ideas]
> As an end user of the application, I need to reduce the total amount of notifications in my life, in order to reduce distraction.

Currently, when having two Outline tabs open and dismissing all notifications in one, they will still show up in the other. If notifications could use the collaborative facilities, their state could be synced via the CRDT. This could allow to dismiss notifications in all browser tabs without the need for a reload, when they were already dismissed in one.

**Comment by tommoor** [comment]
Hi @almereyda – this is already hooked up via websockets and works correctly on cloud. If you're not seeing this behavior then it's likely the main websocket is not connecting correctly on your self-hosted installation.

**Reply by almereyda** [reply]
Okay, thanks. Will investigate.
